greek/ice_bucket_challenge/ice_bucket_challenge.md.txt

```diff
@@ -0,0 +1 @@
+���� ��� �������� �� �������� ���� ��� ���� ������ ���. ��� ��� �� ��������...
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.